### PR TITLE
Feature Development

### PR DESCRIPTION
**Pull request by geraldgsh**
1. Set up a blank HTML document with the appropriate links to your JavaScript and CSS files.

2. Wr0te the functions that hit the API. Specifically to take a location and return the weather data for that location. Console.log() to test.

3. Wrote the functions that process the JSON data(i.e regional time calculation) from the API and return an object with only the data that is required.

4. Set up a simple form to let users input their location and will fetch the weather info (still just console.log() it).

5. Built display for information on webpage!

6. Added styling!

7. Linted and corrected correct code formats.

8. Regular push new feature to github

### CODE REVIEW

**Review by shloch — CHANGES_REQUESTED** [review]
## Great job @geraldgsh 

looks very good

Screen Shot 2020-02-21 at 11 07 38 PM

it's clear you do your job with details. No depth can beat your README...impressive

### [required changes]

- your `.eslintrc.json` file should be same as that of microverse' [ [click here to see](https://github.com/microverseinc/linters-config/blob/master/javascript/.eslintrc.json) ]

- your `.stickler.yml` file should be same as that of microverse' [ [click here to see](https://github.com/microverseinc/linters-config/blob/master/javascript/.stickler.yml)]

Apart of that your weather code is good. You might want to re-run linting after these few changes

Waiting for a last re-review hopefully
Happy coding

**Comment by geraldgsh** [comment]
@shloch Thanks for point out the files. Otherwise I wont realize I've been using those files from Ruby section. Also thanks for providing the screenshot that helped verified local weather information in another location 😄😄

Changes have been made and resubmitted for review

File: readme.md [hunk 1]
```diff
@@ -0,0 +1,516 @@
...
+
+#### Weather APP
+This is a weather app built with Bulma, Webpack, Bootstrap, JS, CSS & HTML.
+
+### Wiki
+
+Checkout our [wikipage](https://github.com/geraldgsh/weather-app/wiki) for more details. 
+
+#### Live Demo
+[Demo]()
```
Comment (MauricioRobayo) [inline readme.md:476]: Don't forge to update the live demo link.

**Review by MauricioRobayo — APPROVED** [review]
Congratulations! All requested changes from the previous code review are fixed.

Status: __Approved__.

I tested the app but no matter which city I tried I constantly got an `invalid city` error or `duplicate city` error. If you need more details to reproduce the error just let me know.

image

Here is the live version I tested: https://cdn.statically.io/gh/geraldgsh/weather-app/354112c6f07932814c72eb79c9d94bdeae23702d/dist/index.html

**Comment by geraldgsh** [comment]
@MauricioRobayo Thanks for pointing the error out. I've debugged it and fixed for production.

**Pull request merged**